docs/gen-docs/kyma_uninstall.md

```diff
@@ -20,20 +23,19 @@ kyma uninstall [flags]
 ### Options
 
 ```
-  -h, --help               help for uninstall
-      --timeout duration   Time-out after which Kyma CLI stops watching the uninstallation progress (default 30m0s)
+  -h, --help               Displays help for the command.
+      --timeout duration   Time-out after which Kyma CLI stops watching the the process of unstalling Kyma. (default 30m0s)
 ```
 
 ### Options inherited from parent commands
 
 ```
-      --kubeconfig string   Path to kubeconfig (default "/Users/i504462/.kube/config")
-      --non-interactive     Do not use spinners
-  -v, --verbose             verbose output
+      --kubeconfig string   Specifies the path to the kubeconfig file. (default "/Users/user/.kube/config")
```

Comment:
This is not true for windows.

The default value is calculated by the kubernetes library as follows: `$HOME_DIR + ".kube/config"`.
To reflect that we can document it something like:
`--kubeconfig string   Specifies the path to the kubeconfig file. (default "$HOME/.kube/config")`

Reply:
I agree - changed :)